src/core/retargeting_engine_tests/python/test_transforms.py

```
@@ -290,12 +290,11 @@ def test_preserves_shape_and_dtype(self):
 
 
 class TestHeadTransform:
-    def _make_head_input(self, position, orientation, is_valid=True, timestamp=100):
+    def _make_head_input(self, position, orientation, is_valid=True):
```

Ruff (0.15.2)

[warning] Missing return type annotation for private function `_make_head_input`

(ANN202)